lib/setupmanagers/hckclient.rb

```diff
@@ -139,21 +151,28 @@ def return_when_client_up
       initialize_client_wait
     end
 
+    def prepare_envirenment
```

Comment:
"Environment" is not a word used elsewhere. Name it `prepare_client` or `prepare_machine`.